### ISSUE TITLE
Device detection bug

### ISSUE THREAD

**Issue by 109km** [mav-rik/facerestore_cf#1]
```
#device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
// You should change the line above to the lines below, cause there could be "mps" device or maybe others.
from comfy import model_management
device = model_management.get_torch_device()
```

**Comment by mav-rik** [comment]
@109km thanks for this finding. Would you mind opening pull request with suggested fix?

**Comment by mav-rik** [comment]
fixed with #6
